support/docker/production/docker-entrypoint.sh

```diff
@@ -11,6 +11,18 @@ cp /app/config/default.yaml /config
 cp /app/support/docker/production/config/custom-environment-variables.yaml /config
 chown -R peertube:peertube /config
 
+# Parsing trust_proxy if available
+if [ ! -z "${PEERTUBE_TRUST_PROXY}" ]; then
+    # We need to replace space by comma
+    TRUST_PROXY="[${PEERTUBE_TRUST_PROXY// /,}]"
+    # Push the trust_proxy conf to production.yaml and remove old config
+    awk -v var="${TRUST_PROXY}" 's{if(/\s*-\s*.*/) next; else s=0}  /trust_proxy:/{print "trust_proxy: " var; s=1; next} 1' /config/production.yaml > /config/temp-production.yaml
+    # We use cat to override some filesystem problem with cp
+    cat /config/temp-production.yaml > /config/production.yaml
+    rm /config/temp-production.yaml
+fi
+chown -R peertube:peertube /config
```

Comment:
then could you remove the previous call to `chown`?

Reply:
I hope too, but I get some strange error about rights when manipulating files inside the volume. I think it could be linked to existing files the entrypoint try to modify.

I have to admit it's mysterious to me this one, perhaps a docker security things about mapped linux UID, but I have to add both chown or sometimes the start failed :/

Reply:
:frowning: that's weird

Reply:
Yes... However I suspect this is problem specific to kubernetes environment (That I used for tests beside docker run) because of its multiple security contexts mechanisms...

While odds, I think it acceptable to be sure right are corrects ^^